crates/symbolicator-service/tests/integration/public_sources.rs

```diff
@@ -0,0 +1,52 @@
+//! Integration tests for some public symbol servers that we want to explicitly fetch from.
+//!
+//! Ideally we should only test each public symbol server we care about only once, as fetching "real"
+//! symbols from the web can be slow and presents a source of flakiness.
+//! FIXME: We currently use the microsoft symbol server all over the place for a couple of tests,
+//! which we should migrate over to use our internal fixtures instead.
```

Comment:
We should really be doing this. Running some of these tests is unnecessarily slow, especially as they depend on your internet connection :-)

Reply:
How should we do this? Download the symbol files we need for the tests and put them in fixtures?

Reply:
I opened an issue to track this: https://github.com/getsentry/symbolicator/issues/1004